### DISCUSSION TITLE
2023.05.29

### DISCUSSION THREAD

**Discussion by chwangmin** [Blue-club/pintos_5#8] [개발 일지]
# 오늘 할 일

김정인
- [x] alarm-clock 구현
- [ ] alarm-clock 블로그 업로드
- [x] priority scheduling 개념 이해 및 공부
- [x] priority scheduling 코딩

장혁
- [ ] 

최광민
- [x] priority scheduling 개념 이해 및 공부
- [ ] priority scheduling 코딩

**Comment by chwangmin** [comment]
#7

**Comment by mywnajsldkf** [comment]
https://github.com/Blue-club/pintos_5/issues/10
